apps/ewallet/lib/ewallet/web/orchestrator.ex

```diff
@@ -0,0 +1,67 @@
+defmodule EWallet.Web.Orchestrator do
+  @moduledoc """
+  This module should be used in every controller to deal with searching, filtering,
```

Comment:
I suggest adding a bit more description of what the module does?

```ex
@moduledoc """
  This module orchestrates the request attributes into search, filter, preload and sort queries, 
  so that this set of features can be easily and consistently applied to controllers.

  This module should be used in every controller to deal with searching, filtering,
  preloading and sorting.
"""
```